backend/kesaseteli/applications/api/v1/views.py

```diff
@@ -209,21 +209,24 @@ def additional_info(self, request, *args, **kwargs) -> HttpResponse:
     def accept(self, request, *args, **kwargs) -> HttpResponse:
         youth_application: YouthApplication = self.get_object().lock_for_update()
 
-        try:
-            serializer = YouthApplicationHandlingSerializer(
-                data=request.data, context=self.get_serializer_context()
-            )
-            serializer.is_valid(raise_exception=True)
-        except ValidationError as e:
-            LOGGER.error(
-                f"Youth application was not changed to accepted state because of "
-                f"validation error. Validation error codes: {str(e.get_codes())}"
-            )
-            raise
+        if settings.DISABLE_VTJ:
```

Comment:
I decided to refactor this a bit more and store `None` i.e. null to the field if `DISABLE_VTJ` is set. Storing an empty JSON object was there because a logic run later sort of required an empty JSON object, but I decided to refactor that bit.